tests/test_cli.py

```diff
@@ -15,8 +15,9 @@ def test_not_authenticated():
     (out_file, out_format, arguments) = cli.parse(args=['-c', 'non-existent', 'domain_search', 'google',
                                                         '--max-length', '100'])
     assert out_format == 'json'
-    assert not arguments.get('user', None)
-    assert not arguments.get('key', None)
+    print(arguments)
```

Comment:
I will remove this in my next MR. I think we only get so many GitHub workflow actions and I would hate to waste it for an errant print statement in the tests.